### DISCUSSION TITLE
how to sync bit type column from mysql to doris

### DISCUSSION THREAD

**Discussion by czxin788** [apache/doris#35501] [A - General / Q&A]
mysql table structure：
```
`IsIn` bit(1) NOT NULL DEFAULT b'1' COMMENT '进出;1-进、0-出',
  `PersonID` bigint(20) NOT NULL DEFAULT '0' COMMENT '负责人',
) ENGINE=InnoDB
```

but  it has errors when create table in doris:
```
Caused by: java.sql.SQLException: errCode = 2, detailMessage = Syntax error in line 1:
...`IsIn` BOOLEAN DEFAULT 'b'1'' COMMENT '进出;1-进、0-出',`Pe...
```

so I modify it  `IsIn` BOOLEAN  NOT NULL DEFAULT  '1' COMMENT '进出;1-进、0-出', and it is ok in doris.

When I insert into data to doris table, it is bad:
```
...20-05-22',323,137,_binary '',0,1785759,1785759,'2020-...
                             ^
Encountered: STRING LITERAL
Expected: ||, COMMA
```

I can look source sql like this:
![image](https://github.com/apache/doris/assets/13861904/07acfa90-3877-4cc6-8fcc-eacc8fda183a)

How can I do to solve  bit type column in doris?

**Comment by yagagagaga** [comment]
Change `_binary ''` to `false` and try again
